package.json

```diff
@@ -56,7 +56,7 @@
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-polyfill": "^6.26.0",
     "babel-preset-env": "^1.6.0",
-    "babel-preset-flow": "^6.23.0",
+    "babel-preset-react": "^6.24.1",
```

Comment:
At version 6 this `babel-preset-react` includes `babel-preset-flow`. However, at version 7, it will be necessary to install both presets